tezos/messages/src/p2p/encoding/current_branch.rs

```diff
@@ -16,7 +16,7 @@ use crate::p2p::encoding::block_header::BlockHeader;
 
 use super::limits::CURRENT_BRANCH_HISTORY_MAX_LENGTH;
 
-pub const HISTORY_MAX_SIZE: u8 = CURRENT_BRANCH_HISTORY_MAX_LENGTH as u8; // 200
+pub const HISTORY_MAX_SIZE: u8 = 200;
```

Comment:
@akoptelov 
could we remove HISTORY_MAX_SIZE now?  CURRENT_BRANCH_HISTORY_MAX_LENGTH is what we want

Reply:
That constant is still used in Tezos code:
```
(* FIXME: this should not be hard-coded *)
let max_locator_size = 200
```

Reply:
@akoptelov 
I know, but we have now:
```
pub const HISTORY_MAX_SIZE: u8 = 200;
```
and 
```
pub const CURRENT_BRANCH_HISTORY_MAX_LENGTH: usize = 200;
```

it is the same, so I just want to remove HISTORY_MAX_SIZE and use CURRENT_BRANCH_HISTORY_MAX_LENGTH instead

Reply:
Not exactly. The `CURRENT_BRANCH_HISTORY_MAX_LENGTH` is actually `1000`.